test/e2e/tests/confirmations/header.spec.js

```diff
@@ -0,0 +1,88 @@
+const { strict: assert } = require('assert');
+const {
+  defaultGanacheOptions,
+  openDapp,
+  unlockWallet,
+  WINDOW_TITLES,
+  withFixtures,
+} = require('../../helpers');
+const FixtureBuilder = require('../../fixture-builder');
+
+describe('Header component', function () {
```

Comment:
If this presents at the top level in the pipeline, should we be explicit and call this `Confirmation Header`?